### ISSUE TITLE
Sprint 2 Aug 5-11 Build/defend

### ISSUE THREAD

**Issue by com1clyf3** [CompletelyFairGames/dwarfcorp#802]
Give players a better reason to build walls. Allow enemies to attack walls on their way to the dwarfs.*
Sprint 2 Aug 5-11 Build/defend

**Comment by com1clyf3** [comment]
EDIT: THIS SPRINT HAS BEEN CHANGED TO MODDING!

**Comment by com1clyf3** [comment]
@Blecki  This get finished?

**Comment by Blecki** [comment]
In progress.

**Comment by mklingen** [comment]
@Blecki update on this?

**Comment by Blecki** [comment]
Yeah, got the structure in place to allow it to download mods in the
background today.

**Comment by mklingen** [comment]
@Blecki, did you ever write documentation? If so I can close this issue and link it in the update.

**Comment by mklingen** [comment]
They make "mana-lamp.png" themselves. It's the icon for the mod. "meta.json" gets placed next to the icon.
